src/qtism/data/rules/BranchRuleCollection.php

```diff
@@ -31,6 +31,20 @@
  */
 class BranchRuleCollection extends QtiComponentCollection
 {
+
+    /** @var bool */
+    private $supportsNonLinearNavigationMode = false;
+
+    public function isAllowedForNonLinearNavigationMode(): bool
```

Comment:
Just a suggestion
```suggestion
    public function allowsNonLinearNavigationMode(): bool
```
or perhaps
```suggestion
    public function isNonLinearNavigationModeAllowed(): bool
```

Reply:
Ok, done. Thanks!